Cron/RetrieveReviews.php

```diff
@@ -50,40 +51,55 @@ public function __construct(
         $this->log = $logger;
         $this->variable = $variable;
 
-        $this->filePath = $this->getFeedUrl();
+        $this->apiKey = $this->getApiKey();
+        $this->locationId = $this->getLocationId();
     }
 
 
-    public function getFeedUrl(){
+    public function getApiKey(){
         return $this->storeManager->getStore()->getConfig(
-            'kiyoh_settings/general/feed_url'
+            'kiyoh_settings/general/api_key'
         );
     }
 
+    public function getLocationId(){
+        return $this->storeManager->getStore()->getConfig(
+            'kiyoh_settings/general/location_id'
+        );
+    }
+
+
     public function processAggregateScores() {
 
-        $this->curl->get($this->filePath);
-        $output = $this->curl->getBody();
+        if (!$this->getLocationId()) {
+            throw new \Magento\Framework\Exception\CronException(__('Location ID missing, please set your location ID.'));
+        }
+
+        if (!$this->getApiKey()) {
+            throw new \Magento\Framework\Exception\CronException(__('API key missing, please set your API key.'));
+        }
 
-        $doc = simplexml_load_string($output);
+        $this->curl->addHeader('X-Publication-Api-Token', $this->getApiKey());
+        $this->curl->get('https://www.klantenvertellen.nl/v1/publication/review/external/location/statistics?locationId=' . $this->getLocationId());
 
-        unset($doc->{'reviews'});
+        $ouput = $this->curl->getBody();
+        $jsonOutput = json_decode($ouput);
```

Comment:
It would be nice if you use the JSON serializer/unserializer in Magento, `\Magento\Framework\Serialize\Serializer\Json`